packages/core/src/Stores/client-store.js

```diff
@@ -765,6 +766,13 @@ export default class ClientStore extends BaseStore {
         return this.account_status?.status?.some(status => status === 'trading_experience_not_complete');
     }
 
+    get is_verified() {
+        return (
+            this.account_status?.authentication?.document?.status === 'verified' &&
+            this.account_status?.authentication?.identity?.status === 'verified'
+        );
+    }
+
```

Comment:
do we need this extra function? cant we use `authentication_status` in client store ?
In the component
 `authentication_status?.document_status !== 'verified' &&  authentication_status?.identity_status === 'verified';`